core/tee/tee_time_generic.c

```diff
@@ -139,3 +139,16 @@ TEE_Result tee_time_set_ta_time(const TEE_UUID *uuid, const TEE_Time *time)
 		return tee_time_ta_set_offs(uuid, &offs, false);
 	}
 }
+
+void tee_time_busy_wait(uint32_t milliseconds_delay)
+{
+	TEE_Time curr, delta, end;
+
+	tee_time_get_sys_time(&curr);
```

Comment:
@jforissier I agree with you but since `tee_time_get_sys_time()` returns a status, I think it should be either handled or explicitly discarded (hence the question marks in my comment).